TLM-SimpleQA-Benchmark/evaluate_response.ipynb

```diff
@@ -0,0 +1,255 @@
+    "                    if (aggregate_metrics[\"accuracy_given_attempted\"] + aggregate_metrics[\"is_correct\"]) > 0\n",
+    "                    else 0\n",
+    "                )\n",
+    "            }\n",
+    "            \n",
+    "            print(f\"Accuracy Given Attempted: {output_d['accuracy_given_attempted']:.3f}\")\n",
+    "            print(f\"F1 Score: {output_d['f1']:.3f}\")\n",
+    "            \n",
+    "            return common.aggregate_results(results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2954184a-2b21-43fa-819a-72ff5b27fb35",
+   "metadata": {},
+   "source": [
+    "In the next cell we show the evaluation for the GPT-4o baseline responses. Here we only run the evaluation on the first 10 examples as a sample, to run it on all examples, set `num_examples` to None)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b015b3c6-8b0d-4f76-9fb9-75175d95d983",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:00<00:00, 12.43it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AGGREGATE METRICS\n",
+      "{'is_correct': 0.3, 'is_incorrect': 0.7, 'is_not_attempted': 0.0, 'is_given_attempted': 1.0, 'accuracy_given_attempted': 0.3}\n",
+      "##################\n",
+      "Accuracy Given Attempted: 0.300\n",
+      "F1 Score: 0.300\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "gpt_4o_baseline = pd.read_csv(\"results/gpt-4o-baseline-responses.csv\")\n",
+    "grading_sampler = ChatCompletionSampler(model=\"gpt-4o\")\n",
+    "\n",
+    "simple_qa_eval = SimpleQAEval(grading_sampler, dataset=gpt_4o_baseline, num_examples=10)\n",
+    "res = simple_qa_eval()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f222b44-91f3-4ae4-810f-60cf525ba74e",
+   "metadata": {},
+   "source": [
+    "Execute the function below to evaluate all datasets generated in the [get_tlm_response.ipynb](get_tlm_response.ipynb) script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "c3f64dbb-8f85-4d04-9401-ccf532428335",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def evaluate_all_datasets():\n",
+    "    dataset_list = [\n",
+    "        \"results/gpt-4o-baseline-responses.csv\",\n",
+    "        \"results/gpt-4o-25-responses.csv\",\n",
+    "        \"results/gpt-4o-80-responses.csv\",\n",
+    "        \"results/gpt-4o-best-responses.csv\",\n",
+    "        \"results/gpt-4o-best-25-responses.csv\",\n",
+    "        \"results/gpt-4o-best-80-responses.csv\",\n",
+    "    ]\n",
+    "\n",
+    "    grading_sampler = ChatCompletionSampler(model=\"gpt-4o-mini\")\n",
```

Comment:
should change this to `gpt-4o`? i believe openai used gpt-4o as the judge

Reply:
Oh good catch, I changed this to teste the code (to save costs) and forgot to change it back. Our blog results are from gpt-4o tho :)